csrankings-a.csv

```diff
@@ -166,7 +166,7 @@ Abu Asaduzzaman,Wichita State University,https://www.wichita.edu/profiles/academ
 Abu Sayed Md. Latiful Hoque,BUET,https://cse.buet.ac.bd/faculty_list/detail/asmlatifulhoque,3-Sb7tMAAAAJ
 Abul Kashem,BUET,https://cse.buet.ac.bd/faculty_list/detail/kashem,NOSCHOLARPAGE
 Abusayeed M. Saifullah,Iowa State University,https://www.cs.iastate.edu/abu,CYwffSUAAAAJ
-Abusayeed Saifullah,Iowa State University,https://www.cs.iastate.edu/abu,CYwffSUAAAAJ
+Abusayeed Saifullah,Wayne State University,https://www.cs.iastate.edu/abu,CYwffSUAAAAJ

```

Comment:
Still says Iowa State on web page: https://www.abusayeedsaifullah.com/

Reply:
I am sorry for the late reply.

Ab is going to join the computer science department at WSU in this coming fall. I noticed that CS ranking updates quarterly, I was wondering when CS ranking updates next time in Oct., Ab should have been working for WSU. The Chair of our department has approved to submit this request.

Please feel free to skip my request if you think it is too early.

Thanks.

Reply:
Ab has updated his affiliation to WSU at https://www.abusayeedsaifullah.com/ . Please check. Thanks.